Cookbook.Infrastructur/Persistance/DbConnectionFactory.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Cookbook.Application.Database;
+using Microsoft.Data.Sqlite;
+using SQLitePCL;
+using System.Data;
+
+namespace Cookbook.Infrastructur.Persistance
+{
+    public class SqliteConnectionFactory : IDbConnectionFactory
+    {
+        private readonly string connectionString =
```

Comment:
Ezt constructorban kene megkapnia. Nezz utana a Configuration-nek, van arra is tok jo support mind a ServiceCollection